src/Accelerate/vImageTypes.cs

```diff
@@ -272,11 +272,11 @@ public static vImageError ConvolveWithBiasARGBFFFF (ref vImageBuffer src, ref vI
 									vImageFlags flags)
 		{
 			if (kernels == null)
```

Comment:
you can also update the `== null` for `is null`
and `!= null` for `is not null`

That was not available when I wrote https://github.com/xamarin/xamarin-macios/wiki/Nullability but it should be added :)

Reply:
> Changing any == null or != null to is null and is not null

well well well :D

Reply:
I wrote down that this PR does the 'is null' and 'is not null' but I guess I lied haha. Thank you for catching my bluff